packages/dd-trace/src/config.js

```diff
@@ -462,6 +462,9 @@ class Config {
     this._setValue(defaults, 'appsec.stackTrace.maxDepth', 32)
     this._setValue(defaults, 'appsec.stackTrace.maxStackTraces', 2)
     this._setValue(defaults, 'appsec.wafTimeout', 5e3) // µs
+    this._setValue(defaults, 'baggageMaxBytes', 8192)
```

Comment:
There wasn't a max before so this is a breaking change.

Reply:
this doesn't apply to the legacy opentracing baggage

Reply:
That could be confusing for a user then, but at least it's non-breaking 👍